issues/Issues.md

```diff
@@ -0,0 +1,60 @@
+# Known Issues
+
+## Windows Issues
+
+####/dev/null git apply issue
+
+New version of git is required to solve this issue.
+
+#### CreateProcess issue
+
+```
+echo  Makefile
+CreateProcess(NULL,echo Makefile,...)
+process_begin: CreateProcess(NULL, echo Makefile, ...) failed.
+Putting child 01DD8B10 (_build) PID 31536216 on the chain.
+Live child 01DD8B10 (_build) PID 31536216
+make (e=2): The system cannot find the file specified.
+Reaping losing child 01DD8B10 PID 31536216
+make: *** [Makefile:323: _build] Error 2
+Removing child 01DD8B10 PID 31536216 from chain.
+```
+
+This issue is due to missing `sh` or `bash` in path.
+Adding `sh` or `bash` will fix the issue.
+
+#### Long paths issue
+
+```
+build   25-Oct-2018 17:17:07      Compiling file: ble_gap_evt_conn.c
+build   25-Oct-2018 17:17:07      ../../../../../../components/serialization/connectivity/codecs/ble/serializers/ble_gap_evt_conn.c:46:10: fatal error: conn_ble_gap_sec_keys.h: No such file or directory
+build   25-Oct-2018 17:17:07       #include "conn_ble_gap_sec_keys.h"
+build   25-Oct-2018 17:17:07                ^~~~~~~~~~~~~~~~~~~~~~~~~
+build   25-Oct-2018 17:17:07      compilation terminated.
+build   25-Oct-2018 17:17:07      make: *** [../../../../../../components/toolchain/gcc/Makefile.common:272:
+```
+
+## macOS Issues
+
+#### Timeout error related to the SEGGER J-Link firmware
+
+On macOS (OS X) there is a known issue with the SEGGER J-Link firmware (that runs on the Debug probe on the board) related to USB packet sizes. This results in the timeout error `Failed to open nRF BLE Driver. Error code: 0x0D` when the serial port is attempted to be opened.
```

Comment:
There are other errors that can occur also, for example:
@511955087972 [0x70000e586000] | [central   ][status] code:PKT_SEND_ERROR message:Error sending packet to target. Code: 0x802a

I think we should document that it is mandatory to disable MSD since the error may surface at varying locations.

Reply:
Will doc.